ci/scripts/install_sccache.sh

```diff
@@ -39,30 +39,30 @@ SCCACHE_URL="https://github.com/mozilla/sccache/releases/download/v$VERSION/scca
 SCCACHE_ARCHIVE=sccache.tar.gz
 
 # Download archive and checksum
-curl -L $SCCACHE_URL --output $SCCACHE_ARCHIVE
-curl -L $SCCACHE_URL.sha256 --output $SCCACHE_ARCHIVE.sha256
+curl -L "$SCCACHE_URL" --output $SCCACHE_ARCHIVE
+curl -L "${SCCACHE_URL}.sha256" --output $SCCACHE_ARCHIVE.sha256
 echo "  $SCCACHE_ARCHIVE" >> $SCCACHE_ARCHIVE.sha256
 
-SHA_ARGS="--check --status"
+SHA_ARGS=(--check --status)
 
 # Busybox sha256sum uses different flags
 if sha256sum --version 2>&1 | grep -q BusyBox; then
   SHA_ARGS="-sc"
```

Comment:
```suggestion
  SHA_ARGS=("-sc")
```

Reply:
Thanks.

What do you think `SHA_ARGS=(-sc)`? Because The previous part `SHA_ARGS=(--check --status)` doesn't use double quote.  If we use quote in this place, It seems better to change `SHA_ARGS=("--check" "--status")`

Reply:
`(-sc)` is fine here.

Reply:
Thanks. I changed to `(-sc)`.